src/tsc/tsc.ts

```diff
@@ -21,4 +21,5 @@ if (ts.sys.setBlocking) {
     ts.sys.setBlocking();
 }
 
+ts.sys.args.push("--skipJSDocParsing");

```

Comment:
My original reaction to this was that this was gross, but, I am not sure what else to do for this, and having a compiler option (though hidden) makes writing tests easier.

Reply:
After sitting down to collect a CPU profile, I definitely think this needs to be configurable from the command line and tsconfig.